SPECS/grpc/generate_source_tarball.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+set -e
+
+SRC_TARBALL="grpc-1.42.0.tar.gz"
```

Comment:
why not using $PKG_VERSION instead of hard coding version number ?

Reply:
made it an empty string.